extract_pipeline.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import pandas as pd
+
+response = requests.get('https://randomuser.me/api/')
+
+data = response.json()
+results = data['results']
+
+normalized_result = pd.json_normalize(results)
+
+#DATA TRANSFORMATION
+#Convert to dataframe
+
+normalized_frame = pd.DataFrame(normalized_result)
+
+
+##DATA TRANSFORMATION: Select Columns
+
+selected_columns = normalized_frame[['name.title', 'name.first', 'name.last', 'cell','email','location.street.number','location.street.name','location.city','location.country',
```

Comment:
This is redundant

Reply:
This was to pick the columns I wanted out of the whole bunch